test/functional/feature_assumeutxo.py

```diff
@@ -152,6 +152,12 @@ def test_invalid_mempool_state(self, dump_output_path):
 
         self.restart_node(2, extra_args=self.extra_args[2])
 
+    def test_invalid_file_path(self):
+        self.log.info("Test bitcoind should fail when file path is invalid.")
+        node = self.nodes[0]
+        path = node.datadir_path / node.chain / "invalid" / "path"
```

Comment:
Got to learn about the functionality of the slash operator in case of Pathlib.Path!
https://docs.python.org/3/library/pathlib.html#operators